registry/repository.go

```diff
@@ -24,6 +24,11 @@ type RepositoryClient struct {
 	remote.Repository
 }
 
+type SignatureManifest struct {
+	Blobs       []artifactspec.Descriptor
```

Comment:
Looks like there is only one descriptor in the blobs, do we want to keep descriptor instead of a slice?

Reply:
One blob is better than a slice.

Reply:
thanks! I'll update it.